Task2/dashboard.js

```diff
@@ -0,0 +1,246 @@
+// fetch('https://reqres.in/api/users')
+// .then(res=>res.json())
+// .then(data=>console.log(data))
+
+// let email=localStorage.getItem('email')?localStorage.getItem('email'):''
+// console.log(email);
+// if(email=='')
+// {
+//     alert('u need to login first');
+//     window.location.href="index.html";
+// }
+
+const root = document.getElementById("root");
+const form = document.getElementById("Form");
+const Post = document.getElementById("btn");
```

Comment:
Avoid variable naming conventions to start with capital letters, instead, use small letters or camelCase.